frontend/src/main/components/Nav/Footer.js

```diff
@@ -44,6 +44,9 @@ export default function Footer() {
           <a data-testid="footer-sticker-link" href="https://www.as.ucsb.edu/sticker-packs">www.as.ucsb.edu/sticker-packs/</a>
           .
         </p>
+        <p>
+          <a href="/privacy">Privacy Policy</a>
```

Comment:
Use react-router’s `Link` over anchor tags to avoid full page reloads.

Reply:
Agreed with Chris.